src/main/java/gbot/Ui.java

```diff
@@ -0,0 +1,44 @@
+package gbot;
+
+/**
+ * The user interface of the chatbot.
+ *
+ * @author Gallen Ong
+ */
+public class Ui {
+    private static final String LINE = "____________________________________________________________";
+
+    /**
+     * Prints a message in between two lines.
+     *
+     * @param message The message to be printed.
+     */
+    public static void print(String message) {
+        System.out.println(LINE);
+        System.out.println(message);
+        System.out.println(LINE);
+    }
+
+    /**
+     * Prints the initial greeting to user.
+     */
+    public void ask() {
+        print("Hello I'm GBot!\nWhat can I do for you?");
```

Comment:
Consider placing the entryGreeting and exitGreeting as constants rather than inside a function. Use functions only when a true verb/action is being performed